$NetBSD$

Disable all sound support for now since it causes
crashes and doesn't work correctly anyway ...

--- widget/src/gtk2/nsSound.cpp.orig	2010-08-06 01:09:23.000000000 +0000
+++ widget/src/gtk2/nsSound.cpp
@@ -138,6 +138,8 @@ nsSound::Init()
 
     mInited = PR_TRUE;
 
+    return NS_OK;
+
     if (!elib) {
         elib = PR_LoadLibrary("libesd.so.0");
     }
